BeehiveCommunity/Frontend_Mentor
# Fund-tracking dashboard

## What does this PR do?

issue  solve   #603

 This PR introduces a new Fund Tracking Dashboard feature that showcases government fund distribution. The dashboard uses HTML, CSS, and JavaScript to create visualizations, including charts and graphs to represent GDP data.

Fixes #(issue)


![desktop-design](https://github.com/user-attachments/assets/5fac8f03-5ec0-4fc3-ae92-93a644a69c25)
## Type of change



- New feature (non-breaking change which adds functionality)


## How should this be tested?

- [Open the Fund Tracking Dashboard page and verify that all graphs and charts display correctly ] Test A
- [Check responsiveness across various devices and screen sizes.] Test B

## Mandatory Tasks
✅Self-reviewed the code.
✅Verified that charts and graphs render and update as expected.
✅Tested functionality on different devices for responsiveness
